client/src/components/assets/InfoCard.jsx

```diff
@@ -3,19 +3,133 @@ import CloseIcon from '@material-ui/icons/Close';
 import axios from 'axios';
 
 const InfoCard = (props) => {
-  const { id, setInfo } = props;
+  const { id, trigger, setTrigger } = props;
+  const IMG_URL = 'https://image.tmdb.org/t/p/w1280';
+
+  const [data, setData] = useState({});
+  const [rating, setRating] = useState();
+  const [status, setStatus] = useState();
+  const [completed, setCompleted] = useState(true);
+  const [title, setTitle] = useState();
+  const [voteAverage, setVoteAverage] = useState();
+  const [pp, setPp] = useState();
+
+  const handleStatusChange = (stat) => {
+    setTitle(data.title);
+    setVoteAverage(data.vote_average);
+    setPp(data.poster_path);
+    if (stat === 'completed') {
+      setCompleted(false);
+      setStatus('completed');
+      return;
+    }
+    setCompleted(true);
+
+    if (stat === 'null') {
+      localStorage.removeItem(id);
+      setStatus('null');
+    } else {
+      setStatus('ptw');
+    }
+    setRating('null');
+  };
+
+  useEffect(() => {
+    if (status === 'ptw' || status === 'completed') {
```

Comment:
ptw is not descriptive enough, either change to a more descriptive variable name or add a comment